src/key_packages/mod.rs

```diff
@@ -211,6 +211,23 @@ impl KeyPackage {
     pub(crate) fn cipher_suite(&self) -> &Ciphersuite {
         self.cipher_suite
     }
+
+    /// Replace the init key in the `KeyPackage` with a random one and return a
+    /// `KeyPackageBundle` with the corresponding secret values
+    pub fn rekey(&self) -> KeyPackageBundle {
```

Comment:
This is a bit of a weird construction. I think it's ok for now, but with #214, we should really update the `KeyPackageBundle` directly, not the `KeyPackage`.

Reply:
Yeah I think we should just start tackling #214. This pops up quite a bit lately.